dev-qt/qtwebengine/qtwebengine-5.15.11_p20230927.ebuild

```diff
@@ -231,6 +231,9 @@ src_configure() {
 	export NINJA_PATH=/usr/bin/ninja
 	export NINJAFLAGS="${NINJAFLAGS:--j$(makeopts_jobs "${MAKEOPTS}" 999) -l$(makeopts_loadavg "${MAKEOPTS}" 0) -v}"
 
+	# Try to force thirdparty components to use user C{,XX}FLAGS as well. bug #652172
+	tc-export_build_env CC CXX
```

Comment:
build_env looks wrong for this? why not normal tc-export?

Reply:
At the time I was thinking in terms of the bug report, so I making CC and CXX are respected at the same time seemed reasonable.

But currently I'll have to look at this further so that I can be sure that this affected it for sure.